doc/classes/VisualScriptLocalVar.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptLocalVar" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Gets a local variable's value.
 	</brief_description>
 	<description>
+		This node returns a local variable's value. "Var Name" must be supplied, with an optional type.
```

Comment:
May be worth adding an example? Not sure if it's important and/or possible. I'd probably learn about VS elsewhere, with more visual docs and experimentation.